### ISSUE TITLE
May I ask the batchsize you set during training and how many Gpus you use to train ?

### ISSUE THREAD

**Issue by UrlyBoy** [midasklr/SSD.Pytorch#12]
Why did I set my batchsize to 32 and train 240 epochs  with single GPU to get only about 72 mAP ?

**Comment by UrlyBoy** [comment]
The training set is VOC 2007+2012 trainval and the test set is VOC2007 test.

**Comment by midasklr** [comment]
batch size 64, 300 epochs, 4*TitanX,
u may try with:
1)warmup
2)lr scheduler
